python/tests/integration/toolbox/test_library_tool.py

```diff
@@ -26,7 +26,7 @@ def get_log_types():
     return [KeyType.LOG, KeyType.LOG_COMPACTED]
 
 
-@pytest.mark.parametrize("lib_type", ["lmdb_version_store", "s3_version_store"])
+@pytest.mark.parametrize("lib_type", ["lmdb_version_store", "s3_version_store", "azure_version_store"])
```

Comment:
Can probably just use `object_version_store` instead of this manual `request.getfixturevalue` indirection.

Reply:
Agree. So I have add a new fixture, similar to `object_version_store` but with `lmdb` additionally for it in later commit.